app/main/posts/common/post-actions.directive.js

```diff
@@ -55,7 +55,7 @@ function PostActionsDirective(
                     // Redirect to list
                     $location.path('/views/data');
                 } else {
-                    $state.go($state.$current, null, { reload: true }); // in favor of $route.reload();
+                    $state.go('list.data', null, { reload: true }); // in favor of $route.reload();
```

Comment:
`$state.reload()` ?

Reply:
no. Because you want to explicitly reload list.data , not whatever state you are in (you could be in list.data.detail, and you don't care about reloading that one.

Reply:
Got it